code/modules/surgery/tools.dm

```diff
@@ -401,6 +450,8 @@
 		clumsy_check = FALSE, \
 	)
 	RegisterSignal(src, COMSIG_TRANSFORMING_ON_TRANSFORM, PROC_REF(on_transform))
+	AddElement(/datum/element/gags_recolorable)
```

Comment:
it's because we're adding the element here twice, once on the parent `/obj/item/scalpel` and again here on the `advanced` subtype. you only need to add the element (and update_icon presumably) once per type. i think retractor has the same issue.